docs/index.rst

```diff
@@ -22,11 +22,16 @@ Currently supported web frameworks and libraries:
 * aioboto3/aiobotocore
 * aiohttp >=2.3
 * boto3/botocore
+* Bottle
 * Django >=1.10
 * Flask
-* httplib
+* httplib/http.client
```

Comment:
Should we use `,` here or put one in a new line? looks like `httplib` and `http.client` are two different client side libraries.

Reply:
httplib and http.client are both native libraries and are used for kind of same low level http clients. httplib is for python2 and http.client is for python3. So in a user's application, it will be either one or another and that's why I think "/" is fine.